ml_engine/online_prediction/noxfile_config.py

```diff
@@ -22,7 +22,7 @@
 
 TEST_CONFIG_OVERRIDE = {
     # You can opt out from the test for specific Python versions.
-    "ignored_versions": ["2.7", "3.8", "3.9", "3.10"],
+    "ignored_versions": ["2.7", "3.6", "3.8", "3.9", "3.10"],
```

Comment:
@alecglassford We have you listed in CODEOWNERS for `ml_engine`, would you know if this change is OK to take?

Reply:
@busunkim96 Hi! I no longer work at the company, so I'm not a good person to answer this question. (You should probably also remove me from `CODEOWNERS`; sorry that I didn't think to do this before I left!)

Reply:
@nicain can you take a look at this?

Reply:
TODO for @nicain find new codeowner for ml_engine, or remove